tooltester/src/main/java/io/dockstore/tooltester/client/cli/ReportCommand.java

```diff
@@ -153,7 +153,11 @@ private void getToolVersionTestResults(ToolVersion toolVersion, String runner, S
                 report.printAndWriteLine(record);
                 if (SEND_LOGS) {
                     if (result.equals(StateEnum.SUCCESS.name())) {
-                        s3Client.createObject(toolId, tag, pipelineNode.getDisplayName(), runner, logContent, epochStartTime);
+                        try {
+                            s3Client.createObject(toolId, tag, pipelineNode.getDisplayName(), runner, logContent, epochStartTime);
+                        } catch (Exception e) {
+                            LOG.error("Could not send logs to s3");
```

Comment:
probably want to do something with the exception so you can narrow down the issue